### PR TITLE
fix: explore org button position and bug fixes

### PR DESCRIPTION
**Pull request by 16-karan**
## WHAT ##
Reset position of explore organisations on navbar as per new landing page design.
Changed edit user close svg from back arrow to close.
Bug: Close button redirects to users page instead of closing notification #134
Bug: Missing space between text and plus (+) icon on the schemas and create organization Page #131
Bug: Updated profile name not displayed on the profile's slider #124
Bug: The length of the description characters differ from the suggested length #138
Resolved type errors faced while working.
Removed commented and unused code.

### CODE REVIEW

File: src/components/Resources/Schema/SchemasList.tsx [hunk 1]
```diff
@@ -197,7 +197,7 @@ const SchemaList = (props: { schemaSelectionCallback: (schemaId: string, schemaD
 									schemaList.map((element, key) => (
 										<div className='p-2' key={key}>
 											<SchemaCard schemaName={element['name']} version={element['version']} schemaId={element['schemaLedgerId']} issuerDid={element['issuerId']} attributes={element['attributes']} created={element['createDateTime']}
-												onClickCallback={schemaSelectionCallback} />
+												onClickCallback={schemaSelectionCallback} className='' />
```
Comment (nishad-ayanworks) [inline src/components/Resources/Schema/SchemasList.tsx:200]: remove this empty className

**Review by nishad-ayanworks — CHANGES_REQUESTED** [review]
Please check comments

File: src/components/Resources/Schema/SchemasList.tsx (hunk 1 above)
Comment (16-karan) [reply src/components/Resources/Schema/SchemasList.tsx]: Removed. Given optional to handle type error.

**Review by nishad-ayanworks — APPROVED** [review]
LG

**Pull request merged**